pkg/agent/flowexporter/connections/connections.go

```diff
@@ -219,9 +225,10 @@ func (cs *ConnectionStore) ForAllConnectionsDo(callback flowexporter.ConnectionM
 	return nil
 }
 
-// Poll calls into conntrackDumper interface to dump conntrack flows
+// Poll calls into conntrackDumper interface to dump conntrack flows. In a dual-stack setup, there will be two connsLens
+// get returned.
```

Comment:
suggestion: Poll calls into conntrackDumper interface to dump conntrack flows. It returns the number of connections  for each address family, as a slice. In dual-stack clusters, the slice will contain 2 values (number of IPv4 connections first, then number of IPv6 connections).

Reply:
Updated.